xbmc/powermanagement/PowerManager.cpp

```diff
@@ -298,6 +305,14 @@ void CPowerManager::OnWake()
   g_weatherManager.Refresh();
 
   CServiceBroker::GetPVRManager().OnWake();
+  //TODO set Stack if its missing?
```

Comment:
let's keep it simple and drop stacking. nobdy seems to know what this ancient piece of code does. if stack does not work on resume, nobody cares. if so, we leave it those to implement it.

Reply:
I guess we will drop playlist for the same reason

Reply:
I want to consider playlists. For music playlists are more common than single files.

Reply:
I wouldn't drop stacking because it's still needed for older material. It dates back to the old ages of movies being split onto multiple files (cd1, cd2, etc.) There are two kinds of stacks,  "normal" ones that have avi or mkv files, where you can seamlessly play the next parts, and then "dvd" or "iso" stacks where each part has a separate menu. This used to be the case with double-sided DVDs like "Jesus of Nazareth (1977)".

Reply:
@Voyager1 don't get this wrong. Nobody wants to drop stacking, it's just not considered for this sleep/resume case. Means no regression on stacking. The reason why it's not considered here is lack of knowledge. If you have time you are very welcome to add the missing pieces.

Reply:
@FernetMenta thanks. Right now not a lot of time but as of next week yes I can take a look at it. I agree to keep it out for now.